Dockerfile

```
@@ -0,0 +1,7 @@
+FROM node:22.17.0 
+
+WORKDIR /web
+ADD . .
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)